testing/util/attestation.go

```diff
@@ -78,7 +78,7 @@ func GenerateAttestations(
 			if err != nil {
 				return nil, err
 			}
-			headState = state.BeaconState(genState)
+			headState = genState
```

Comment:
I see another `headState = state.BeaconState(genState)`. Want to fix that as well?

Reply:
Done. Thanks!